components/basecomponents/KidsTranslationContainer.tsx

```diff
@@ -1,53 +1,86 @@
 import React from 'react';
-import Image from 'next/legacy/image';
+import { useAtomValue } from 'jotai';
+
+/** Components */
 import { KidsTranslation } from './KidsTranslation';
+import KioskDictionaryCardImage from './KioskDictionaryGame/KioskDictionaryCardImage';
+
+/** Hooks, Types, Utils, etc. */
 import { useLanguage } from 'utils/useLanguageHook';
-import { AudioPlayer } from 'utils/AudioPlayer';
-import { useTranslation } from 'next-i18next';
-import { Phrase } from '../../utils/getDataUtils';
-
-interface KidsTranslationContainerProps {
-  phrase: Phrase;
-  imageUrl: string | null;
-  id?: string;
-  searchText?: string;
-}
-
-/**
- *  Displays list of translations in opened collapse component
- *
- * @returns
- */
-export const KidsTranslationsContainer = ({ phrase, imageUrl, id }: KidsTranslationContainerProps): JSX.Element => {
+import { Platform, KidsTranslationContainerProps } from '@types';
+import { currentPlatformAtom, dictionaryAudioPlayAtom, dictionaryActivePhraseAtom } from './Kiosk/atoms';
+import { Language, LanguageEnum } from 'utils/locales';
+
+const ACTIVE_STYLE = {
+  [LanguageEnum.CZECH]: 'czech-sound !bg-kiosk-red',
```

Comment:
Can this not be achieved using native Tailwind without introducing global css classes?